imports/plugins/included/default-theme/client/styles/navbar.less

```diff
@@ -340,3 +338,7 @@ html:not(.rtl) .rui.navbar .languages .dropdown-menu {
   border: solid rgba(129, 122, 122, 0.62) 2px;
   box-sizing: content-box !important;
 }
+
+.dropdown-toggle button.rui.btn.btn-default.flat {
```

Comment:
Selector seems way too specific. Can it not be simplified?

Reply:
It can be, but I chose it to be that specific so that nothing breaks in some other place later. As such the selector can be just `.dropdown-toggle button`.

Should I modify it?

Reply:
@mikemurray Can you reply to @Akarshit here?

Reply:
@mikemurray Hey Mike, could you reply to @Akarshit here? Second call.

Reply:
@Akarshit could you use`.rui.navbar .dropdown-toggle .btn`? Just so it's not specific to button states?

Reply:
@mikemurray Ok will do.